src/test/java/org/terasology/logic/health/HealEventTest.java

```diff
@@ -124,9 +123,27 @@ public void healModifyEventTest() {
                     event.subtract(5);
                     event.multiply(2);
                 });
-        // expected = ( initial:10 + 10 - 5 ) * 2 = 30
+        // Expected value is ( initial:10 + 10 - 5 ) * 2 == 30
         player.send(new DoHealEvent(10));
         assertEquals(30, player.getComponent(HealthComponent.class).currentHealth);
     }
 
+    @Test
+    public void healNegativeTest() {
```

Comment:
Uh oh! This test actually fails for me! Expected 40, got 50.

Not worried about it though, merging anyway :-)